api/tests/command_test.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import call_command
```

Comment:
I'm not totally sure this test is getting run when i run `make test` -- is there a special command you were using to run it? (I'm mostly basing this on how long the tests take to run which seems to quick for it to be running the full import task of 1220 records)

That also brings up another question which is whether we really want to run that full import on every test run. I wonder if it's possible to run for a slimmer set of data in the tests so it stays faster.